src/mpid/ch4/netmod/ofi/ofi_comm.c

```diff
@@ -59,3 +59,10 @@ int MPIDI_OFI_mpi_comm_free_hook(MPIR_Comm * comm)
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_OFI_MPI_COMM_FREE_HOOK);
     return mpi_errno;
 }
+
+int MPIDI_OFI_mpi_comm_set_hints(MPIR_Comm * comm, MPIR_Info * info)
```

Comment:
nitpick - remove `_mpi_` from these and associated APIs. That convention is supposed to be for functions that correspond 1-to-1 with MPI functions.

Reply:
I wasn't even consistent with myself. I apparently only called it `_mpi_` inside the CH4 API... 🤦